shared/js/textChatView.jsx

```diff
@@ -198,11 +198,26 @@ loop.shared.views.chat = (function(mozL10n) {
       var entriesClasses = classNames({
         "text-chat-entries": true
       });
+      var hideHelloWelcome = this.props.messageList.length > 0 ||
+        loop.shared.utils.isDesktop();
+
+      if (this.props.messageList.length > 0) {
+        var msgEntry = this.props.messageList[0];
```

Comment:
Is Room name message always the first element of the array? I will suggest to use messageList.filter instead to check if we have a CHAT_CONTENT_TYPES.ROOM_NAME element. @Standard8